### ISSUE TITLE
log response length

### ISSUE THREAD

**Issue by yoshuawuyts** [yoshuawuyts/http-ndjson#3]
need to override the `res` object to pull it off though

**Comment by yoshuawuyts** [comment]
``` js
const resStream = require('response-length-stream')
const stdout = require('stdout-stream')
const pump = require('pump')
const http = require('http')

http.createServer((req, res) => {
  const sink = resStream(res)
  const httpLogger = httpNdjson(req, res)

  sink.on('end', () => {
    const size = sink.size()
    httpLogger.setSize(size)
    res.setHeader('Content-Length', size)
  })

  pump(httpNdjson(req, res), stdout)
  pump(req, sink)
}).listen()

```

**Comment by yoshuawuyts** [comment]
released as `v2.1.0`
